bin/a0-ext

```diff
@@ -52,5 +54,31 @@ prog
             });
           });
       });
+  })
+  .command('build:client', 'Compile your React Frontend into an Auth0 Extension Client')
```

Comment:
Will all extensions have to be updated to have a `client:build` step? We need to make sure this fits into the CI/CD plans cc @shawnmclean